dynamiqs/utils/utils.py

```diff
@@ -350,20 +349,21 @@ def ptrace(x: Tensor, keep: int | tuple[int, ...], dims: tuple[int, ...]) -> Ten
         (..., m, m): Density matrix (with `m <= n`).
 
     Raises:
+        ValueError: If the input tensor is not a ket, bra or density matrix.
         ValueError: If `dims` does not match the input tensor shape, or if `keep` is
         incompatible with `dims`.
```

Comment:
```suggestion
        ValueError: If `dims` does not match the input tensor shape, or if `keep` is
            incompatible with `dims`.
```
Otherwise the second line is not rendered.